tests/small_input.y4m

```diff
@@ -0,0 +1,7 @@
+YUV4MPEG2 W64 H64 F25:1 Ip A16:9 C420jpeg XYSCSS=420JPEG
```

Comment:
What is the source of this file?

Reply:
objective-1-fast blue_sky scaled down to 64x64

Reply:
(To clarify: I tried generating a test file in Rust using random values, but kept failing to get the headers right, so I opted to let ffmpeg do the work)

Reply:
Sounds good.